etc/spgw_u.conf

```diff
@@ -57,83 +57,114 @@ SPGW-U =
         S1U_S12_S4_UP :
         {
             # S-GW binded interface for S1-U communication (GTPV1-U) can be ethernet interface, virtual ethernet interface, we don't advise wireless interfaces
-            INTERFACE_NAME         = "@SGW_INTERFACE_NAME_FOR_S1U_S12_S4_UP@";  # STRING, interface name, YOUR NETWORK CONFIG HERE
+            INTERFACE_NAME         = "{{ env["SGW_INTERFACE_NAME_FOR_S1U_S12_S4_UP"] if "SGW_INTERFACE_NAME_FOR_S1U_S12_S4_UP" in env.keys() else 'eth0' }}";  # STRING, interface name, YOUR NETWORK CONFIG HERE
             IPV4_ADDRESS           = "read";                                    # STRING, CIDR or "read to let app read interface configured IP address
             #PORT                   = 2152;                                     # Default is 2152
             SCHED_PARAMS :
             {
-                #CPU_ID       = 2;
-                SCHED_POLICY = "SCHED_FIFO"; # Values in { SCHED_OTHER, SCHED_IDLE, SCHED_BATCH, SCHED_FIFO, SCHED_RR }
-                SCHED_PRIORITY = @THREAD_S1U_PRIO@;
-                POOL_SIZE = @S1U_THREADS@; # NUM THREADS
+                #CPU_ID        = 2;
+                SCHED_POLICY   = "SCHED_FIFO"; # Values in { SCHED_OTHER, SCHED_IDLE, SCHED_BATCH, SCHED_FIFO, SCHED_RR }
+                SCHED_PRIORITY = {{ env["THREAD_S1U_PRIO"] if "THREAD_S1U_PRIO" in env.keys() else '80' }};
```

Comment:
@rdefosse If possible let set the default values for threading as below

  threadsS1uUl: "1"
  threadsSgiDl: "1"
  threadsSx: "1"
  threadsSgiPrio: 98
  threadsS1uPrio: 88
  threadsSxPrio: 88